gcp/appengine/frontend3/src/styles.scss

```diff
@@ -642,10 +647,13 @@ dl.vulnerability-details,
     .cta {
       margin: 32px 0;
       text-align: center;
+
+      a {
```

Comment:
A cleaner way to do this might be to apply `display: flex;` to the .cta and add a `gap: 2px;` property or something like that

Ref: https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-gap-row-gap-column-gap